docs/CreateAKSClusterForScalarDL.md

```diff
@@ -17,36 +17,50 @@ When deploying ScalarDL Ledger, you must:
 * Create the AKS cluster by using Kubernetes version 1.21 or higher.
 * Configure the AKS cluster based on the version of Kubernetes and your project's requirements.
 
-### Note
+{% capture notice--info %}
+**Note**
 
 For Byzantine fault detection in ScalarDL to work properly, do not deploy your application pods on the same AKS cluster as the ScalarDL Ledger deployment.
 
+{% endcapture %}
+
+<div class="notice--info">{{ notice--info | markdownify }}</div>
```

Comment:
@josh-wong By the way, is there any way to preview this kind of special style somewhere or somehow (how it will be shown in the actual doc site) before merging PRs?

Reply:
@jnmt Currently, previewing the docs site is a little cumbersome since the docs don't live in the same repository as where the docs site lives. For now, the method that I use to preview docs is to deploy the docs site locally with the changes and then check that the styles appear as expected.

In the near future, I'd like to have our docs based in the docs repositories so that anyone can preview docs (among other benefits).

Reply:
> In the near future, I'd like to have our docs based in the docs repositories so that anyone can preview docs (among other benefits).

Sounds nice! It's no problem since you can preview and review changes as a final check, but nice to have it for other ones' easy review with better styles of the docs site. Thank you!